docs/checkers/index.md

```diff
@@ -2124,6 +2124,30 @@ Example:
 
 ```
 
+By default, there are no Input / output functions ([`input`], [`open`] and [`print`]) allowed.
```

Comment:
Don't capitalize "input".

Also, remove the spaces around the / (and below as well)